docs/cli/commands.md

```diff
@@ -822,3 +822,44 @@ Which will print:
   }
 }
 ```
+
+You can use a multi-line query. The shell will check for a semi-colon to know when you're done.
+
+```bash
+tableland>select 
+tableland>    id, name
+tableland->   FROM
```

Comment:
Should this be `tableland> FROM` without the `-` in `tableland-> FROM`?

Reply:
Oops! Yep, fixed the typo. Ty.